### DISCUSSION TITLE
Freesync/G-Sync/VRR toggle along with potentially other controls for panel settings for connected displays

### DISCUSSION THREAD

**Discussion by Kickskii** [seerge/g-helper#1243] [Ideas]
might be nice to have it in software as an option

**Comment by seerge** [comment]
@Kickskii thats something you have to set in your gpu driver settings like nvidia control panel.

It’s absolutely not related to g-helper

**Reply by Kickskii** [reply]
the reason why i was suggesting this feature is cause G-Helper already lets you adjust Panel refresh rate and varybright
having the option to contol freesync/enhanced sync would be a great addition since most panels these days do come with
